apps/app/src/views/routes/Settings/index.tsx

```diff
@@ -109,7 +113,8 @@ export const Settings = () => {
 
   return (
     <Page header="Settings">
-      {loading ? (
+      {/* Show loading spinner until the correct tab index has been resolved */}
+      {loading || tabIndex == null ? (
```

Comment:
`tabIndex == null` isn't strict equals so this should be totally fine, but since the type of tabIndex is `<number | undefined>` why not `tabIndex === undefined`?

Reply:
I always do `== null`. I probably was taught once. I forget why